compiler/env/OMRCompilerEnv.cpp

```diff
@@ -140,3 +142,40 @@ OMR::CompilerEnv::initializeTargetEnvironment()
 #endif
 
    }
+
+
+void
+OMR::CompilerEnv::initializeRelocatableTargetEnvironment()
+   {
+
+   // Target processor bitness
+   //
+#ifdef TR_TARGET_64BIT
+   relocatableTarget.setBitness(TR::bits_64);
+#elif TR_TARGET_32BIT
+   relocatableTarget.setBitness(TR::bits_32);
+#else
+   relocatableTarget.setBitness(TR::bits_unknown);
+#endif
+
+   // Initialize the relocatable target CPU by querying the host processor
+   //
+   relocatableTarget.cpu = TR::CPU::detectRelocatable(TR::Compiler->omrPortLib);
+
+   // Target major operating system
+   //
+#if HOST_OS == OMR_LINUX
```

Comment:
I don't know whether there is something called `TARGET_OS`, but if there is, it should be used in preference of `HOST_OS` to allow us to do cross compilation in the future.

Reply:
It doesn't seem we have `TARGET_OS`. If there are cross compilation targets in the future, they will probably appear in the parameters and they can overwrite HOST_OS.